test/fixture/build.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs').promises
+
+const mod = () => ((types, b64) => ({
+  replacer(key) {
+    var val = this[key]
+    return ArrayBuffer.isView(val) ? {$d: b64.encode(new Uint8Array(val))} : val
+  },
+  reviver: (key, val) =>
+      val === null && val !== 'object' ? val:
+      val.$d ? new Blob(b64.decode(val)):
+      val
```

Comment:
looks like the generated output (`fixture.test.js`) was edited and committed. so when you run based on the pulled file, it works, but if you rebuild fixtures, it breaks. fixing in my PR https://github.com/transcend-io/conflux/pull/54/files